GeneratorInterface/Core/interface/TaggedProtonHepMCFilter.h

```diff
@@ -0,0 +1,25 @@
+#ifndef __TAGGEDPROTONHEPMCFILTER__
+#define __TAGGEDPROTONHEPMCFILTER__
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "GeneratorInterface/Core/interface/BaseHepMCFilter.h"
+
+class TaggedProtonHepMCFilter : public BaseHepMCFilter {
+private:
+  const int proton_PDGID_ = 2212;
+  const int neutron_PDGID_ = 2112;
+  float OneOverbeamEnergy_;
+  double xiMin_ = 0.02;
+  double xiMax_ = 0.2;
+  double comEnergy_ = 13600.0;
+  int nProtons_ = 2;
```

Comment:
```suggestion
  const double OneOverbeamEnergy_;
  const double xiMin_;
  const double xiMax_;
  const int nProtons_;
```
i.e.:
- `OneOverbeamEnergy_` is used in calculations with all doubles, I don't see why to keep it as a float
- `comEnergy_` is only used to define  `OneOverbeamEnergy_`, there is no need to keep it as a class member
- all class members initialized in the initialization list would overwrite these initial values: no need to initialize them twice, then
- all class members initialized in the initialization list and never modified afterwards can be declared `const`